batch/batch/front_end/front_end.py

```diff
@@ -1394,26 +1396,17 @@ async def close_batch(request, userdata):
         (batch_id,),
     )
     if record:
-        await _commit_update(batch_id, 1, db)
-    return await _close_batch(app, batch_id, user, db)
-
-
-async def _commit_update(batch_id: int, update_id: int, db: Database):
-    await db.execute_and_fetchone(
-        '''
-UPDATE batch_updates
-SET `committed` = TRUE
-WHERE batch_id = %s, update_id = %s
-''',
-        (batch_id, update_id),
-    )
+        await _commit_update(app, batch_id, 1, user, db)
+    return web.Response()
 
 
-async def _close_batch(app: aiohttp.web.Application, batch_id: int, user: str, db: Database):
+async def _commit_update(app: web.Application, batch_id: int, update_id: int, user: str, db: Database):
     client_session: httpx.ClientSession = app['client_session']
     try:
         now = time_msecs()
-        await db.check_call_procedure('CALL close_batch(%s, %s);', (batch_id, now), 'close_batch')
```

Comment:
we should delete the close_batch procedure and remove it from estimated-current.sql

Reply:
You can't do this until a separate PR after all the changes are in. The old batch driver will still need that stored procedure.